service/arm_templates/node_setup.json

```diff
@@ -179,22 +177,27 @@
             "version": "[parameters('osImageVersion')]"
           },
           "osDisk": {
-            "name": "[variables('osDiskName')]",
-            "vhd": {
-              "uri": "[concat('http://', parameters('storageAccountName'), '.blob.core.windows.net/vhds/', variables('osDiskName'), '.vhd')]"
+            "name": "[concat(parameters('clusterID'), '-', parameters('prefix'), '-', parameters('osDisk').name)]",
+            "caching": "[parameters('osDisk').caching]",
+            "managedDisk": {
+              "storageAccountType": "[parameters('osDisk').storageAccountType]"
             },
-            "caching": "ReadWrite",
             "createOption": "FromImage"
           },
-          "dataDisks": [
+          "copy": [
```

Comment:
what this "copy" means?

Reply:
This is loop in ARM templates. Value of `name` is `dataDisks`, then you have # of copies in `count`, then `input `is the content. You can use `copyIndex(name)` to obtain the index of the current loop.